lib/bracket-matcher.coffee

```diff
@@ -29,6 +29,25 @@ class BracketMatcher
     else
       @pairedCharacters = @defaultPairs
 
+  excludePairs: (excludePairs) ->
```

Comment:
Disclaimer: I am unfamiliar with the structure of bracket-matcher.

It looks like this logic is being duplicated in bracket-matcher-view...would it be possible to unify the two?